RecoPixelVertexing/PixelVertexFinding/plugins/gpuClusterTracksByDensity.h

```diff
@@ -59,7 +59,7 @@ namespace gpuVertexFinder {
     if (verbose && 0 == threadIdx.x)
       printf("booked hist with %d bins, size %d for %d tracks\n", hist.nbins(), hist.capacity(), nt);
 
-    assert(nt <= hist.capacity());
+    assert(nt <= (unsigned)hist.capacity());
```

Comment:
It is typically safer to cast the unsigned into a signed, rather than the opposite.
It is probably not the case here, but would `hist.capacity()` be a negative number, once casted into an unsigned it would result in a huge positive number, which will make the assert fail.

Reply:
alright I'll do the opposite I think the bottom of this is a int (as the program resolves to)
https://cmssdt.cern.ch/lxr/source/HeterogeneousCore/CUDAUtilities/interface/FlexiStorage.h#0042